From: Patrick Brunschwig <patrick@enigmail.net>
Date: Tue, 13 Feb 2018 18:01:05 +0100
Subject: fixed editing drafts (or edit as new message): don't fail if uri is
 null

(cherry picked from commit 7ceb98cc7acc0989109aa6c3f2bfcf177cf078b9)
---
 package/uris.jsm | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/package/uris.jsm b/package/uris.jsm
index e2f8b5e..ae3674e 100644
--- a/package/uris.jsm
+++ b/package/uris.jsm
@@ -114,6 +114,8 @@ var EnigmailURIs = {
     // mailbox:///...?number=4455522&part=1.1.2&filename=test.eml&type=application/x-message-display&filename=test.eml
     // imap://user@host:port>UID>some>path>10?header=filter&emitter=js&examineEncryptedParts=true
 
+    if (!url) return null;
+
     EnigmailLog.DEBUG("uris.jsm: msgIdentificationFromUrl: url.pathQueryRef=" + ("path" in url ? url.path : url.pathQueryRef) + "\n");
 
     let msgNum = "";
